src/openapi/peridio-admin-openapi.yaml

```diff
@@ -3829,6 +3829,9 @@ components:
       maxLength: 128
     custom-metadata:
       type: object
+      maxLength: 1048570
+      description: |
+        A JSON object that informs the metadata that will be associated with this artifact version's binaries when they are included in bundles.
```

Comment:
This description is going to be a problem, e.g.:

<img width="941" alt="Screenshot 2024-06-26 at 13 46 59" src="https://github.com/peridio/peridio-docs/assets/868014/9a158f66-fe26-4f90-bd6a-463259d3e90e">

This was the final self-correction I made on the CLI PR for language reference.